pset-iterators.js

```diff
@@ -115,6 +135,15 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter(president => {
+  let tookOffice = parseInt(president.took_office.substring(0,4))
+  if(tookOffice >= 1850 && tookOffice <= 1900){
+    return president
+  }
+})
+
+console.log(presidentsBetween1850and1900)
+
```

Comment:
@nmikhol Good job here using the filter method to return all president who took office between `1850` and `1900`.Good job using the parseInt method to turn the substring of the took_office value to a number then comparing the `tookOffice` in your conditional to see if `tookOffice` lands between 1850 and 1900.

Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L120)